### ISSUE TITLE
Are you looking for a new maintainer?

### ISSUE THREAD

**Issue by d11wtq** [kien/ctrlp.vim#591]
Hi,

I can't help but notice there haven't been any commits for approaching 12 months now, nor has there been any response to pull requests for a considerable amount of time. I understand that open source projects take their toll at times and it's difficult to garner enough motivation to work on them (I've been there myself and handed over popular projects to new maintainers). In the interest of keeping ctrlp.vim moving forward, would you be interested in passing it over to a new maintainer, or at least adding a collaborator with commit access to this repository and retain the ownership yourself?

Not particularly putting myself forward to do this (although I'd be happy to). I just thought somebody should start the conversation as ctrlp.vim has a huge user-base and I'm sure there's community interest in keeping the project active.

Cheers,

Chris

**Comment by sassanh** [comment]
Guys any news about @kien ? I hope he's alright.

**Comment by Shougo** [comment]
Please read the README.

**Comment by sassanh** [comment]
Yeah I'm using the new repository for a while. But all of a sudden while using ctrlp I felt it's really useful and I remembered about this thread and came here to see any news about kien. Btw, the merge commit for README file is a good sign.

**Comment by WhyNotHugo** [comment]
FWIW, if you contact GitHub support, on cases like this, they'll change the cannonical/"root" repository -- so that new forks say "forked from `ctrlpvim/ctrlp.vim`", rather than this one.

This'd really help discoverability via forks quite a bit.
